pkg/vmimage/data/etc/rc.d/rc.local

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -ex
 
 # https://github.com/openshift/openshift-azure/issues/1454
-for pos in /var/log/journald.pos/worker0/storage.json /var/log/openshift-audit.pos; do
+for pos in /var/run/td-agent/*.pos/*/storage.json /var/run/td-agent/*-cursor.json; do
```

Comment:
The upstream bug looks like it was fixed in May, do we even need this hack anymore?

Reply:
the fluentd version we'll use (from the yum repos) doesn't have that patch yet afaict